contracts/consensus/Consensus.sol

```diff
@@ -528,6 +598,18 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
         );
 
         decisions[currentMetablock.metablockHash] = _committeeDecision;
+
+        CommitteeI committee = committees[currentMetablock.metablockHash];
+
+        emit MetablockCommitteeDecided(
+            address(committee),
+            currentMetablock.metablockHash,
+            _metachainId,
+            committeeSize,
+            _committeeDecision,
+            committee.quorum(),
+            uint256(currentMetablock.round)
```

Comment:
- `committee.quorum()` will be fixed and can be evaluated from committee size. it's not needed.

- `currentMetablock.round` doesn't need to be emitted here. Round are fixed enums.  Handler can update the round as `CommitteeDecided` based on event name.

Reply:
Updated